159356-1702-Extramural/capstone
# Remove duplicate script

Sorry missed this when doing on MR for Alyan.

We had duplicate code in client.js and the setup.js which was setting up duplicate sockets.

The PR removes the duplicate front end script.

To test: 
- Start an new game
- Confirm that you see: 

![screen shot 2017-08-12 at 1 49 38 pm](https://user-images.githubusercontent.com/1098068/29236799-59fcfc2a-7f65-11e7-9bf9-40dd30be2372.png)


(Previously it thought you were two people and started the game immediately)
